core/edit_text_file.py

```diff
@@ -16,13 +16,7 @@
     for name, file_name in {
         "abbreviations": "abbreviations.csv",
         "additional words": "additional_words.csv",
```

Comment:
Can we make customize look at the .talon-list files first?

Reply:
Added support for talon list files